src/components/FilterHeaderCategories.js

```diff
@@ -92,7 +92,7 @@ const styles = StyleSheet.create({
     backgroundColor: categoriesFilterButtonBgColor,
     alignItems: "center",
     width: 44,
-    height: 44 * PixelRatio.getFontScale(),
+    height: 44 * Math.min(1.25, PixelRatio.getFontScale()),
```

Comment:
With `44 * Math.min(1.25, PixelRatio.getFontScale())`, would you mind creating a helper with a meaningful name? It is not immediately obvious what the intention is here and a named function could help that.

Reply:
Yes, that's a good idea.